client/commonFramework/add-loop-protect.js

```diff
@@ -5,7 +5,10 @@ window.common = (function(global) {
   } = global;
 
   loopProtect.hit = function hit(line) {
-    var err = `Error: Exiting potential infinite loop at line ${line}.`;
+    var err = 'Error: Exiting potential infinite loop at line ${line}. You can '  
+               + 'disable loop protection by adding "//noprotect" to your code; ho'
```

Comment:
* Multiline strings should have the operator at the end of the line. 
* Indenting here looks weird. It should be indented only two spaces.
* Break the strings at the spaces instead of in mid word.